opensrp-app/src/main/java/org/smartregister/login/interactor/BaseLoginInteractor.java

```diff
@@ -185,6 +188,13 @@ private void remoteLoginWith(String userName, String password, LoginResponse log
         scheduleJobsPeriodically();
         scheduleJobsImmediately();
 
+        // Initialize the P2P library here
+        String teamId = getUserService().getUserDefaultTeamId(loginResponse.payload());
+
+        if (!TextUtils.isEmpty(teamId)) {
+            P2PLibrary.init(new P2PLibrary.Options(userName, new P2PSyncAuthorizationService(teamId)));
```

Comment:
Add this logic also to `org.smartregister.login.interactor.BaseLoginInteractor#localLoginWith`

Reply:
Are there login by a person on a different team on the same device? Is this allowed in either core or any of the host applications?

Reply:
no, user must be on the same team

Reply:
If user is of different team then we may need to trigger a deletion of data transfer history

Reply:
Cool, needed to know if I need to handle the fact the teamId might also change